python_modules/libraries/dagster-k8s/dagster_k8s/ops/k8s_job_op.py

```diff
@@ -211,18 +211,26 @@ def k8s_job_op(context):
     wait_for_job(
         job_name=job_name,
         namespace=namespace,
-        wait_timeout=config.get("timeout", 0),
+        wait_timeout=timeout,
+        start_time=start_time,
     )
 
-    pod_names = get_pod_names_in_job(job_name, namespace=namespace)
+    pods = wait_for_job_to_have_pods(
+        job_name,
+        namespace,
+        wait_timeout=timeout,
+        start_time=start_time,
```

Comment:
whats up with `start_time` here? we want to apply timeouts relative to the start of the overall process not each step? Felt confused reading the code figuring this out

Reply:
That's what I would expect as a user, yeah - overall timeout for the op, not a timeout for whatever random individual steps make up the op